easybuild/easyconfigs/i/impi/impi-2021.3.0-intel-compilers-2021.3.0.eb

```diff
@@ -0,0 +1,16 @@
+name = 'impi'
+version = '2021.3.0'
```

Comment:
@SebastianAchilles I just noticed there's a `2021.3.1` release too, we should probably use that instead?

Reply:
I didn't see it on https://software.intel.com/content/www/us/en/develop/articles/oneapi-standalone-components.html
Do you have a link?

Reply:
That's... interesting. It was released on July 23rd 2021.

Download link: https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/18016/l_mpi_oneapi_p_2021.3.1.315_offline.sh

Reply:
Thank you @boegel
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/13639/commits/8d84fad2ef6a057f6d58df6252197be9fe3242bf